### PR TITLE
Circular Edit Button On Mobile

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/\[slug\]**](https://workshops-h45nrl9qe.hackclub.dev/[slug]) |
|:-:|
|<a href="https://workshops-h45nrl9qe.hackclub.dev/[slug]">Screenshot of /[slug]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/a5bwbK7iTxZNnpGLbmaBHS9W1cMD6_AFlJX_riteGx_1MyUXss99owsbtyq_Hm7IQ_gVR_VmUzLvg78VX7lJYw.png">(view full size)</a> |







Commit 335d4257d79368be6a998a01ec4f6e95988ce27b (https://workshops-h45nrl9qe.hackclub.dev).

**Comment by cjdenio** [comment]
It isn't circular for me :thonk:

Screen Shot 2022-04-18 at 11 34 00

File: pages/[slug]/index.js [hunk 1]
```diff
@@ -64,7 +64,8 @@ const Page = ({ slug, data, html, locales }) => {
             backgroundColor: 'background',
             svg: {
               mr: [0, 2]
-            }
+            },
+            py: ['20px', 2]
```
Comment (jessicard) [inline pages/[slug]/index.js:68]:
```suggestion
            py: ['15px', 2]
```

Makes a nice round button!
Screen Shot 2022-04-18 at 4 29 36 PM

Comment (cjdenio) [reply pages/[slug]/index.js:68]: I've achieved a perfect button with `14px` of vertical padding 🙇

Screen Shot 2022-04-18 at 5 30 37 PM

**Pull request closed without merging**